fix/code clean

KatarzynaStefanowiczPagepro opened the pull request:
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=660) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/JoinColony/colonyCDapp?pullRequest=660) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/JoinColony/colonyCDapp?pullRequest=660) it.</sub>

KatarzynaStefanowiczPagepro commented:
- Fixed checkbox - added mark to a checked checkbox
- Standardized CSS rules
- Used appearance prop in Numeral instead of adding not needed overriding styles - - Added missing into format messages
- Fixed HTML tags
- Added missing hovers
- Used TitleLabel 
